packages/wrangler/src/d1/utils.ts

```diff
@@ -14,6 +14,12 @@ export function getDatabaseInfoFromConfig(
 			d1Database.database_id &&
 			(name === d1Database.database_name || name === d1Database.binding)
 		) {
+			if (!d1Database.database_name) {
+				throw new UserError(
+					`${name} bindings must have a "database_name" field`
```

Comment:
It looks like currently they're required to have a `database_id` field only—should this error be changed?